### ISSUE TITLE
Getting Pool details from AFA Unity fails (dellemc_unity_storagepool.py)

### ISSUE THREAD

**Issue by Piaskun** [dell/ansible-unity#9]
There is bug in dellemc_unity_storagepool.py, line 234:

```
            is_fast_vp_enabled = api_response._get_property_from_raw(
                'pool_fast_vp').is_schedule_enabled
```

For AFA Unities (ones that I can test on) 'pool_fast_vp' is set to None, so this is failing internally with:
`ERROR: 'NoneType' object has no attribute 'is_schedule_enabled'`

and Ansible playbook is reporting error (due to way error is being catch):
`Storage pool not found - Creation of storage pool is not allowed through Ansible module`

FIX: It seems like module is trying to pull just True/False so **api_response._get_property_from_raw('pool_fast_vp')** (no attribute **is_schedule_enabled**) should be used, and translated to true/false, if 'null/none' is not expected in Ansible module output.

**Comment by anupamaloke** [comment]
@Piaskun,  thank you for sharing this. It seems like you might have a fix for this issue already. If yes, then would you be willing to submit a PR?

**Comment by anupamaloke** [comment]
Closed by #38
